### ISSUE TITLE
Update EthMagicians multi-sig with more keyholders and policy

### ISSUE THREAD

**Issue by lrettig** [ethereum-magicians/scrolls#24]
Hello,

With a substantial, generous donation from Cindicator + BokkyPooBah, in addition to many smaller donations, the existing FEM wallet currently has 43.39 ETH, an amount that can be expected to grow over time. I am very grateful to @jpitts and @bmann for managing this process up to now, but I think we can probably all agree that decentralizing governance and management of these funds is in everyone's interest and in the long-term interest of the organization.

As one point of comparison, the #EIP0 wallet has 5-6 keyholders and (IIRC) a 3-of-5 release policy. While I don't know what the optimal number is, I suspect something closer to ~5 keyholders is probably appropriate.

This conversation may be better served on the FEM forum but I'll kick off here with a few questions:

1. How many total keyholders should there be?
1. How many need to sign a tx to release it?
1. Who should those keyholders be?
1. How do we "elect" those keyholders?

As a separate but related topic, it would also be nice to discuss how to further decentralized governance of these funds over the long term by e.g. moving in the direction of a DAO.

Thanks!

Related: #22

**Comment by pet3r-pan** [comment]
Just to clarify: 

I did not raise it in light of the Magicians but directly instead asked as an individual for a donation to help make and fund bounties in the community. The funds were sent on the trust that I use them with the initial purpose and in the best interests of the community. I do not see the funds as part of the current FEM wallet and any previous function of the Magicians.

That given said, I would love to see this to be the beginning of a Magicians Bounty Fund that funds community management + other event efforts :)

**Comment by pet3r-pan** [comment]
But this is also nonetheless a very important issue that has now come to our attention!

**Comment by bmann** [comment]
@pet3r-pan let's talk source of funds in the forum https://ethereum-magicians.org/t/discussion-about-bounties-and-in-general-governance-of-donated-funds/1485/6

In short: if it isn't meant to be for the Magicians as a whole, I personally don't want it in the wallet.

@lrettig let's call this issue the "upgrade the EthMagicians wallet multisig" -- governance as a whole is more of a longer discussion item and I think we should discuss it at length. I've added it as topics of discussion under [Meta Magicians ring](https://hackmd.io/DaJhrasLQteUk3IwX5bQAg?view#5-Meta-Magicians-Ring) -- I can't edit issues, so perhaps you or @jpitts can change that.

The Infrastructure page on the wiki documents a couple bits of things of who owns what https://github.com/ethereum-magicians/scrolls/wiki/Infrastructure

I would not call the donation substantial -- I mean, it's very welcome, and it doubles what was in there before, but there are many many many many bodies with way more funding power that are really slowly doling it. If we have one more Council _without_ a venue sponsor like we partnered with Status on, we'll need most of that to make a deposit.

**Comment by bmann** [comment]
We need to make a payment to Josef #32 to cover catering.

I'm experimenting with [Multis](http://multis.co) -- a web UI on top of Gnosis Wallet that should make it easier for us to add people and give them access, as well as take notes on transactions.

I have invited @jpitts to the interface as the other keyholder.

On this topic, we discussed at [Meta Magicians in Prague](https://ethereum-magicians.org/t/breakout-session-2-meta-magicians/1750) and I'm going to turn a number of those items into Github issues and call for consensus on them.
